packages/pwa/app/pages/account/wishlist/partials/wishlist-secondary-button-group.jsx

```diff
@@ -43,14 +45,9 @@ const WishlistSecondaryButtonGroup = ({productListItemId, list, onClick = noop})
     const handleItemRemove = async () => {
         onClick(variant.id)
         try {
-            await customerProductLists.deleteCustomerProductListItem(list, {id: productListItemId})
-            showToast({
-                title: formatMessage({defaultMessage: 'Item removed from wishlist'}),
-                status: 'success'
-            })
-        } catch (err) {
-            console.error(err)
-            showToast({
+            await wishlist.removeItem(productListItemId)
```

Comment:
Reminder: There should be a toast confirming the action after the item has been removed

Reply:
good catch!!

for context, i attempted to DRY the duplicate toast from many components, and extract that into the hook like:
```jsx
const useWishlist = (options = {}) => {
    const enableToast = options?.enableToast

    const {formatMessage} = useIntl()
    const toast = useToast()
    const wishlist = useCustomerProductList(PWA_DEFAULT_WISHLIST_NAME, options)

    if (!enableToast) {
        return wishlist
    }

    const onError = () =>
        toast({
            title: API_ERROR_MESSAGE,
            status: 'error'
        })

    const onSuccessAdd = ({quantity}) => {
        toast({
            title: `${quantity} items added to wishlist`,
            status: 'success',
            action: (
                <Button as={Link} variant="link" to="/account/wishlist">
                    View
                </Button>
            )
        })
    }

    const onSuccessRemove = () => {
        toast({
            title: 'Item removed from wishlist',
            status: 'success'
        })
    }

    return {
        ...wishlist,
        addItem: withAsync(wishlist.addItem, {
            onSuccess: onSuccessAdd,
            onError
        }),
        updateItem: withAsync(wishlist.updateItem, {
            onError
        }),
        removeItem: withAsync(wishlist.removeItem, {
            onSuccess: onSuccessRemove,
            onError
        }),
        removeItemByProductId: withAsync(wishlist.removeItemByProductId, {
            onSuccess: onSuccessRemove,
            onError
        })
    }
}
```

But unfortunately due to the intl limitation (intl not accessible on top level hooks), i had to revert my changes and put the toast back to all the components, and i missed this component.